Patch-Source: https://gitlab.alpinelinux.org/alpine/abuild/-/merge_requests/131
--
From 6c43d67a053be0476384fcf8252c07fa2a235a87 Mon Sep 17 00:00:00 2001
From: psykose <alice@ayaya.dev>
Date: Thu, 30 Dec 2021 21:48:50 +0000
Subject: [PATCH] newapkbuild: refine default cmake format

MinSizeRel is a better recommended default- of course it can still be
overriden.
the dot is redundant- we already specify -B and so the dot is implicitly
cwd.
---
 newapkbuild.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/newapkbuild.in b/newapkbuild.in
index c58d519..63b387d 100644
--- a/newapkbuild.in
+++ b/newapkbuild.in
@@ -65,8 +65,8 @@ build_cmake() {
 		-DCMAKE_INSTALL_PREFIX=/usr \\
 		-DCMAKE_INSTALL_LIBDIR=lib \\
 		-DBUILD_SHARED_LIBS=True \\
-		-DCMAKE_BUILD_TYPE=None \\
-		\$CMAKE_CROSSOPTS .
+		-DCMAKE_BUILD_TYPE=MinSizeRel \\
+		\$CMAKE_CROSSOPTS
 	cmake --build build
 __EOF__
 }
-- 
GitLab

